feat: remove return from predict and _local_predict (#27)
Substra/substrafl: CHANGELOG.md

### Changed

> - The `predict` and `_local_predict` methods of the Torch Algorithms does not need to return anything anymore.

Reviewer: ```suggestion
- return statement of both `predict` and `_local_predict` methods from Torch Algorithms.
```

Reviewer: under a ### Removed section
